core/java/test/org/openda/exchange/timeseries/TimeSeriesTest.java

```diff
@@ -470,18 +467,18 @@ public static void testTimeSeries2() {
 
    }
 
-   public static void testTimeSeriesSelection_2() {
+   public void testTimeSeriesSelection_2() {
```

Comment:
Codacy found an issue: [The JUnit 3 test method name 'testTimeSeriesSelection_2' doesn't match 'test[A-Z0-9][a-zA-Z0-9]*'](https://app.codacy.com/gh/OpenDA-Association/OpenDA/pullRequest?prid=5102854)